# contracts/test/FHERC20NativeWrapper_Harness.sol  ->  contracts/test/FHERC20NativeWrapper_Harness.fsol
# upstream 5138cb8 .. ported 4cefd22

diff --git a/contracts/test/FHERC20NativeWrapper_Harness.sol b/contracts/test/FHERC20NativeWrapper_Harness.fsol
index f63f6bf..359e25a 100644
--- a/contracts/test/FHERC20NativeWrapper_Harness.sol
+++ b/contracts/test/FHERC20NativeWrapper_Harness.fsol
@@ -2,9 +2,9 @@
 pragma solidity ^0.8.25;
 
 import { IERC20Metadata } from "@openzeppelin/contracts/interfaces/IERC20Metadata.sol";
-import { IWETH } from "../interfaces/IWETH.sol";
-import { FHERC20 } from "../FHERC20/FHERC20.sol";
-import { FHERC20NativeWrapper } from "../FHERC20/extensions/FHERC20NativeWrapper.sol";
+import { IWETH } from "../interfaces/IWETH.fsol";
+import { FHERC20 } from "../FHERC20/FHERC20.fsol";
+import { FHERC20NativeWrapper } from "../FHERC20/extensions/FHERC20NativeWrapper.fsol";
 
 contract FHERC20NativeWrapper_Harness is FHERC20NativeWrapper {
     constructor(
